### ISSUE TITLE
Electron - On Premise doesn't work on intranet

### ISSUE THREAD

**Issue by dershodan** [ubergrape/grape-web-client#371]
#### Step to reproduce

1. Have local on-premise instance  (not reachable from outside / the internet)
2. Try to login using a DNS record that points to local address
3. doesn't connect, shows no error

![image](https://cloud.githubusercontent.com/assets/2463469/23521509/9a91e46e-ff7f-11e6-8c26-ccdd56aa30b4.png)

#### Expected result

Should connect

#### What happened instead

Didnt connect

#### Browser and OS version

Mac with electron 2.10.0

Customer **DSIRF** has this problem right now.

Since the DNS resolution works in normal browsers we must assume, the bug is in electron.

**Comment by sk7** [comment]
needs more to make it easily reproducable since team frontend doesn't have an on-prem vagrant (takes too much time). e.g. provide VPN access to our windows infrastructure a provide a DNS entry for the on-prem instance and configure the on-prem instance to allow this domain name

**Comment by sk7** [comment]
@r900 you already set something up with your domain?

**Comment by r900** [comment]
yes but i'm not sure if we want to use `grape.fucked-up.net` as our (if only internal) solution ;)

**Comment by sk7** [comment]
heh, yeah maybe grape-internal.ubergrape.com or something
